Makefile

```diff
@@ -0,0 +1,15 @@
+help:
```

Comment:
I'd suggest we use goreleaser (https://goreleaser.com) to build the `readium` binary, which we can also run in gh actions to generate new releases.
I guess this is OK as a temporary solution.

Reply:
It requires installing a third-party dependency to build the project locally though. However to generate GH release it definitely makes sense.